docs/Agent/user-guide/man_page.md

```diff
@@ -300,31 +232,31 @@ pbench-register-tool-set, Register the specified toolset
 Register all the tools in the specified toolset.
 
 **OPTIONS**  
- --toolset  
+ `--toolset`  
 Available tool sets from /opt/pbench-agent/config/pbench-agent.cfg:
 
 - heavy
 - legacy
 - light
 - medium
 
---remotes  
+`--remotes`  
 Single remote host, a list of remote hosts (comma-separated, no spaces) or an "at" sign ("@") followed by a filename. In this last case, the file should contain a list of hosts and their (optional) labels. Each line of the file should contain a hostname, optionally followed by a label separated by a comma (","); empty lines are ignored, and comments are denoted by a leading hash, or pound ("#"), character.
 
---labels  
+`--labels`  
 Where the list of labels must match the list of remotes.
 
---help  
+`--help`  
 Show this message and exit.
 
 ---
 
-### pbench-register-tool-trigger
+##### pbench-register-tool-trigger
 
 ---
 
 **NAME**  
-pbench-register-tool-trigger, Registers the tool trigger
+`pbench-register-tool-trigger`, Registers the tool trigger
 
 **SYNOPSIS**  
 `pbench-register-tool-trigger[-C][-g][--start-trigger][--stop-trigger][--help]`
```

Comment:
This synopsis doesn't show any of the options taking values, as is the case with a number of the tools; however, `pbench-register-tool` and some others show the values, which seems appropriate.

A quick look at some Linux man pages suggests that, for options which don't change the mode of operation, the synopsis just uses `[OPTIONS]` and relegates the formats to the `OPTIONS` section of the page.